src/Uuid/Uuid.php

```diff
@@ -305,40 +303,31 @@ public static function uuid_unparse($uuidAsBinary)
             return null;
         }
 
-        $dataAsArray = unpack('H*', $uuidAsBinary);
-        $data = $dataAsArray[1];
-
-        if (32 !== \strlen($data)) {
+        if (16 !== \strlen($uuidAsBinary)) {
             return false;
         }
 
-        $uuid = sprintf('%s-%s-%s-%s-%s',
-            substr($data, 0, 8),
-            substr($data, 8, 4),
-            substr($data, 12, 4),
-            substr($data, 16, 4),
-            substr($data, 20, 12)
-        );
+        $uuidAsBinary = bin2hex($uuidAsBinary);
```

Comment:
Overriding the variable `$uuidAsBinary` to contain the uuid as hex looks wrong to me.

Reply:
updated